### PR TITLE
Fixes #9981: Add node policy mode api documentation

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/9981

### CODE REVIEW

File: src/api_v2/node.md [hunk 1]
```diff
@@ -595,6 +595,70 @@ HTTP/1.1 200 OK
...
+    @apiDescription This API allows to update the node settings.
+
+    @apiExample Initial content
+# Given the "data.json" JSON file with content:
+{ "properties": [
+  { "name": "env_type"    , "value": "production" },
+  { "name": "shell"       , "value": "/bin/sh" },
+  { "name": "utf-8 poetry", "value": "ᚠᛇᚻ᛫ᛒᛦᚦ᛫ᚠᚱᚩᚠᚢᚱ᛫ᚠᛁᚱᚪ᛫ᚷᛖᚻᚹᛦᛚᚳᚢᛗ" }
+] 
+, "policyMode" : "audit"
```
Comment (fanf) [inline src/api_v2/node.md]: you should start with enforce to let the user understand that "policyMode=default" actually changed something in the example call.

File: src/api_v2/node.md [hunk 2]
```diff
@@ -595,6 +595,70 @@ HTTP/1.1 200 OK
...
+        "name": "env_type",
+        "value": "production"
+      },
+      {
+        "name": "shell",
+        "value": "/bin/sh"
+      },
+      {
+        "name": "utf-8 poetry",
+        "value": "ᚠᛇᚻ᛫ᛒᛦᚦ᛫ᚠᚱᚩᚠᚢᚱ᛫ᚠᛁᚱᚪ᛫ᚷᛖᚻᚹᛦᛚᚳᚢᛗ"
```
Comment (fanf) [inline src/api_v2/node.md]: That value should not be here after removing it

File: src/api_v2/node.md [hunk 3]
```diff
@@ -595,6 +595,70 @@ HTTP/1.1 200 OK
...
+{
+  "action": "updateNode",
+  "id": "4db088c8-d849-4f08-bfa9-ac96a22d461a",
+  "result": "success",
+  "data": {
+    "policyMode": "audit",
+    "properties": [
+      {
+        "name": "env_type",
+        "value": "production"
```
Comment (fanf) [inline src/api_v2/node.md]: that value should not be here after removing it.

**Comment by VinceMacBuche** [comment]
Commit modified

**Pull request merged**